### PR TITLE
EPMEDU-773 UI adjustment with latest designs

### PR DESCRIPTION
**Pull request by n3gbx**
1. Fixed paddings to match designs
2. Redesign app bar to match designs
3. Removed buttons elevation
4. Updates string resources
5. Fixed a bug with ime padding on Profile screens
6. Redesigned bottom bar FAB, added coloured shadow
7. Added phone number field placeholder: xxx xx-xx-xx
8. Made birthdate field not editable
9. Changed birthdate format: d MMM, yyyy
10. Added date constraints to date picker
11. Added country flag to phone field
12. Replaced MoreOption with ListItem composable

[Video](https://user-images.githubusercontent.com/17089577/183601490-6e400394-4b12-423a-be0c-58d21e475a00.mp4)

### CODE REVIEW

File: feature/login/src/main/java/com/epmedu/animeal/login/profile/presentation/FinishProfileScreenUI.kt [hunk 1]
```diff
@@ -59,29 +59,33 @@ internal fun FinishProfileScreenUI(
     }
 
     Scaffold(
-        modifier = Modifier.fillMaxSize(),
-        topBar = {
-            TopBar(title = stringResource(id = R.string.profile_title))
-        }
+        modifier = Modifier.fillMaxSize().imePadding(),
```
Comment (Fileek) [inline feature/login/src/main/java/com/epmedu/animeal/login/profile/presentation/FinishProfileScreenUI.kt]: Please put modifier methods on different lines

File: library/foundation/src/main/java/com/epmedu/animeal/foundation/button/AnimealSecondaryButton.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    enabled: Boolean = true,
+    onClick: () -> Unit,
+) {
+    Button(
+        modifier = modifier
+            .height(60.dp)
+            .fillMaxWidth(),
+        colors = ButtonDefaults.buttonColors(
+            backgroundColor = DisabledButtonColor,
+            contentColor = MaterialTheme.colors.onPrimary,
```
Comment (Fileek) [inline library/foundation/src/main/java/com/epmedu/animeal/foundation/button/AnimealSecondaryButton.kt:29]: According to designs text color should be white for both light and dark themes. You could probably change the onPrimary color in the dark color palette to white

File: library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/AnimealAlertDialog.kt [hunk 3]
```diff
@@ -28,7 +28,7 @@ fun AnimealAlertDialog(
 ) {
     AlertDialog(
         modifier = Modifier.padding(24.dp),
-        shape = RoundedCornerShape(32.dp),
+        shape = RoundedCornerShape(24.dp),
```
Comment (Fileek) [inline library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/AnimealAlertDialog.kt]: Why 24 if designs have 30?

File: library/foundation/src/main/java/com/epmedu/animeal/foundation/button/AnimealSecondaryButton.kt (hunk 2 above)
Comment (n3gbx) [reply library/foundation/src/main/java/com/epmedu/animeal/foundation/button/AnimealSecondaryButton.kt:29]: Dark theme designs are not final. I think there is no need to work on it now until they got finalised

File: feature/login/src/main/java/com/epmedu/animeal/login/profile/presentation/FinishProfileScreenUI.kt (hunk 1 above)
Comment (n3gbx) [reply feature/login/src/main/java/com/epmedu/animeal/login/profile/presentation/FinishProfileScreenUI.kt]: Updated

File: library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/AnimealAlertDialog.kt (hunk 3 above)
Comment (n3gbx) [reply library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/AnimealAlertDialog.kt]: Updated

**Review by Fileek — APPROVED** [review]

**Review by egorikftp — APPROVED** [review]

**Pull request merged**